plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -1266,4 +1383,131 @@ private ColumnMapping uuidColumnMapping()
                 (resultSet, columnIndex) -> javaUuidToTrinoUuid((UUID) resultSet.getObject(columnIndex)),
                 uuidWriteFunction());
     }
+
+    private static class StatisticsDao
+    {
+        private final org.jdbi.v3.core.Handle handle;
```

Comment:
why full import? I don't see any other `Handle` in this class.

Reply:
thanks, intellij's fault.